### PR TITLE
Added Lookups test

### PR DESCRIPTION
**Pull request by tannoa2**
@jlongever @zyoung51 @uppalk1 @RackHD/corecommitters @keedya

### CODE REVIEW

File: test/tests/lookups_tests.py [hunk 1]
```diff
@@ -62,4 +69,65 @@ def check_lookup_id(self):
...
+        """ Negative Testing POST """
+        #Validate that a POST for a lookup with same id as an existing one gets rejected
+        try:
+            Lookups().api1_1_lookups_post(self.lookup)
+        except Exception,e:
+           assert_equal(500,e.status, message = 'status should be 505')
+
+    @test(groups=['check-patch-lookup'],depends_on_groups=['check-lookups-query','check-post-lookup','check-post-lookup-negativeTesting'])
+    def patch_lookup(self):
+        """ Testing PATCH """
```
Comment (jlongever) [inline test/tests/lookups_tests.py]: be a little more descriptive.

File: test/tests/lookups_tests.py [hunk 2]
```diff
@@ -62,4 +69,65 @@ def check_lookup_id(self):
...
+        Lookups().api1_1_lookups_id_patch(self.id,self.patchedNode)
+
+        #validate that the node element has been updated
+        Lookups().api1_1_lookups_get(self.lookup.get("macAddress"))
+        rsp = loads(self.__client.last_response.data)
+        assert_equal(str(rsp[0].get("node")),self.patchedNode.get("node") )
+
+    @test(groups=['check-delete-lookup'], depends_on_groups=['check-lookups-query','check-post-lookup','check-patch-lookup'])
+    def delete_lookup(self):
+        """ Testing DELETE """
```
Comment (jlongever) [inline test/tests/lookups_tests.py]: be a little more descriptive.

File: test/tests/lookups_tests.py [hunk 3]
```diff
@@ -62,4 +69,65 @@ def check_lookup_id(self):
                 rsp = self.__client.last_response
                 assert_equal(200, rsp.status, message=rsp.reason)
 
+    @test(groups=['check-post-lookup'],depends_on_groups=['check-lookups-query'])
+    def post_lookup(self):
+        """ Testing POST """
```
Comment (jlongever) [inline test/tests/lookups_tests.py]: be a little more descriptive.

File: test/tests/lookups_tests.py [hunk 4]
```diff
@@ -62,4 +69,65 @@ def check_lookup_id(self):
...
+        #Validate the content
+        Lookups().api1_1_lookups_get(self.lookup.get("macAddress"))
+        rsp = loads(self.__client.last_response.data)
+        assert_equal(str(rsp[0].get("ipAddress")),str(self.lookup.get("ipAddress")))
+        assert_equal(str(rsp[0].get("macAddress")),str(self.lookup.get("macAddress")))
+        assert_equal(str(rsp[0].get("node")),str(self.lookup.get("node")))
+
+    @test(groups=['check-post-lookup-negativeTesting'],depends_on_groups=['check-lookups-query','check-post-lookup'])
+    def post_lookup_negativeTesting(self):
+        """ Negative Testing POST """
```
Comment (jlongever) [inline test/tests/lookups_tests.py]: be a little more descriptive.

**Comment by jlongever** [comment]
address the test descriptions, otherwise :+1:

**Comment by jlongever** [comment]
UNSTABLE -

**Comment by jlongever** [comment]
test this please

**Comment by jlongever** [comment]
test this please

**Comment by jlongever** [comment]
test this please

**Comment by tannoa2** [comment]
test this please

**Comment by jlongever** [comment]
test this please

**Comment by jlongever** [comment]
+1

**Pull request merged**